.github/workflows/release.yaml

```diff
@@ -0,0 +1,36 @@
+# Release workflow: Pushing a tag triggers this workflow,
+# which uses the name of the tag as the container tag/version
+# that goes into GHCR.
+name: Release
+
+# Build and publish Docker image upon pushing a tag
+on:
+  push:
+    tags:
+      - "v*"
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Right here we need to add these to fix the annotated tag, and to verify that it really is an annotated tag in the end:

```
    # actions/checkout@v2 breaks annotated tags by converting them into
    # lightweight tags, so we need to force fetch the tag again
    # See: https://github.com/actions/checkout/issues/290
    - name: Repair tag
      run: git fetch -f origin ${{ github.ref }}:${{ github.ref }}
    - name: Verify tag is annotated
      run: if test x$(git for-each-ref ${{ github.ref }} | awk '{print $2}') = xtag; then /bin/true; else echo "\"${{ github.ref }}\" does not look like an annotated tag!"; /bin/false; fi
```

Reply:
Done.